### DISCUSSION TITLE
Cleanup User's in Database

### DISCUSSION THREAD

**Discussion by mfenton01** [opendcim/openDCIM#1423] [Q&A] [answered]
Over time databese has users no longer needed .  Is there a best way to remove ?

**Comment by samilliken** [comment, marked as answer]
Yeah, wait for 23.03 to be released.

 

From: mfenton01 ***@***.***> 
Sent: Saturday, August 19, 2023 8:45 PM
To: opendcim/openDCIM ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [opendcim/openDCIM] Cleanup User's in Database (Discussion #1423)

 

Over time databese has users no longer needed . Is there a best way to remove ?
